onadata/libs/utils/logger_tools.py

```diff
@@ -353,7 +353,8 @@ def create_instance(username,
             instance.save()
 
         instance = DuplicateInstance()
-
+    except DataError as error:
```

Comment:
This `except` block here is irrelevant, we are not raising a specific exception. I will recommend removing it.

Reply:
Done